app/services/key_rotator/hmac_fingerprinter.rb

```diff
@@ -14,7 +14,7 @@ def rotate(user:, pii_attributes: nil, profile: nil)
 
     def rotate_email_fingerprint(user)
       ee = EncryptedAttribute.new_from_decrypted(user.email)
-      user.update_columns(email_fingerprint: ee.fingerprint)
+      UpdateUser.new(user: user, attributes: { email_fingerprint: ee.fingerprint }).call
```

Comment:
oh huh that's interesting, sorry I merged before I saw this comment.

I feel like using an update method that does not run validations or callbacks makes sense in *some* situtations, like this one, but perhaps not all? 

In general in any rails app I would expect records to be updated with `update` or `update!`, but I could be convinced otherwise, I supposed.